ctapipe/io/dl1eventsource.py

```diff
@@ -42,8 +42,8 @@ class DL1EventSource(EventSource):
     method. An event equals an ArrayEventContainer instance.
     See ctapipe.containers.ArrayEventContainer for details.
 
-    Attributes:
-    -----------
+    Parameters
```

Comment:
No, Attributes is correct for a class, Parameters is for functions / methods. The only error was the colon I think.

Reply:
ups - my bad